data/processors/merge.py

```diff
@@ -43,12 +44,14 @@ def add_translatable_str(value: Union[str, list[Any], dict[str, Any]]) -> Any:
 
 def add_coordinates(data, path):
     """
-    Merge coordinates from the yaml file at path on top of the given data.
+    Merge coordinates from yaml files placed at path on top of the given data.
+    (Merging happens in alphanumeric order, so later files would overwrite earlier files)
     This operates on the data dict directly without creating a copy.
     """
-    yaml_data = load_yaml(path)
+    for fname in os.listdir(path):
+        yaml_data = load_yaml(os.path.join(path, fname))
```

Comment:
I like the `pathlib.Path` syntax of iterating through files a bit more, as this way this becomes more readbale:
```suggestion
    for file in Path(path).iterdir():
        yaml_data = load_yaml(file)
```
(and either extracting the path from the file or modifing/looking at `load_yaml`)